assets/src/scss/components/_buttons.scss

```diff
@@ -12,23 +12,25 @@
 .btn,
 .wp-block-button a,
 .wp-block-file .wp-block-file__button {
+  --button-- {
+    font-family: $roboto;
+    text-align: center;
+    font-weight: bold;
+    border-radius: 4px;
+    padding: 0 30px;
+    transition-property: color, background-color, border-color;
+    transition-duration: 150ms;
+    transition-timing-function: linear;
+  }
   display: inline-block;
   position: relative;
-  font-family: $roboto;
   font-size: $font-size-sm;
```

Comment:
Why is this font-size not a variable? 🤔

Reply:
Because there's also the `button-small` and `button-large` classes. And the donate button also sets a custom size, but only for `large-and-up`. So I'm not sure if it makes sense to also add it to the "base" button class.